src/python/grpcio/grpc/experimental/aio/_server.py

```diff
@@ -13,34 +13,47 @@
 # limitations under the License.
 """Server-side implementation of gRPC Asyncio Python."""
 
-from typing import Text, Optional
 import asyncio
+from concurrent.futures import Executor
+from typing import Any, Optional, Sequence, Text
+
 import grpc
-from grpc import _common
+from grpc import _common, _compression
 from grpc._cython import cygrpc
 
+from ._typing import ChannelArgumentType
+
+
+def _augment_channel_arguments(base_options: ChannelArgumentType,
+                               compression: Optional[grpc.Compression]):
+    compression_option = _compression.create_channel_option(compression)
+    return tuple(base_options) + compression_option
+
 
 class Server:
     """Serves RPCs."""
 
-    def __init__(self, thread_pool, generic_handlers, interceptors, options,
-                 maximum_concurrent_rpcs, compression):
+    def __init__(self, thread_pool: Optional[Executor],
+                 generic_handlers: Optional[Sequence[grpc.GenericRpcHandler]],
+                 interceptors: Optional[Sequence[Any]],
+                 options: ChannelArgumentType,
+                 maximum_concurrent_rpcs: Optional[int],
+                 compression: Optional[grpc.Compression]):
```

Comment:
Maybe I'm a bit late on that discussion or it's worth it to preserve the current API, but I'm wondering how this `compression` parameter fits on the idea of having support for levels of compression eventually (are they currently supported in Python?, any plan?)

How good or bad will be naming the `compression` parameter as `default_compression_algorithm`??

Reply:
Do you mean as a way to signal clients, that this would be a "fallback compression algorithm" in case it's set, but none is provided on the calls?
I think it's still better to keep the signature compatible with the synchronous version, to make it simpler for users.
Perhaps a docstring could help address that issue?

Reply:
@rmariano  Updated docstring:
```
compression: An element of grpc.compression, e.g.
        grpc.compression.Gzip. This compression algorithm will be used for the
        lifetime of the server unless overridden by set_compression. This is an
        EXPERIMENTAL option.
```

@pfreixes The name `compression` still intuitive enough to let user know it is setting the default compression algorithm. The options I mentioned are more or less advanced usage.